packages/jellyfish-api-core/__tests__/category/net/getNodeAddresses.test.ts

```diff
@@ -0,0 +1,27 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { TestingGroup } from '@defichain/jellyfish-testing'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+
+describe('Network on masternode', () => {
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+  const group = TestingGroup.create(2)
+
+  beforeAll(async () => {
+    await container.start()
+    await group.start()
+  })
+
+  afterAll(async () => {
+    await container.stop()
+    await group.stop()
+  })
```

Comment:
```suggestion
  const group = TestingGroup.create(2)

  beforeAll(async () => {
    await group.start()
  })

  afterAll(async () => {
    await group.stop()
  })
```

When you're using `TestingGroup.create(numberOfContainer)`, you essentially setup multiple containers. Hence you don't need to setup anymore containers.

Accessing container at index 0:
`group.get(0).rpc.getNodeAddresses(1)`

Accessing container at index 1:
`group.get(1).rpc.getNodeAddresses(1)`